documentation/devonfw-ide-introduction.asciidoc

```diff
@@ -1,12 +1,68 @@
+* *Automated* +
+Automate the setup and update, avoid manual steps and mistakes.
+* *Simple* +
+KISS (Keep It Small and Simple), no native installers that globally mess your OS or tool-integration that break with every release. Instead, use templates and simple shell scripts.
+* *Configurable* +
+You can change the link:configuration.asciidoc[configuration] depending on your needs. Furthermore, the link:settings.asciidoc[settings] contain configuration templates for the different tools (see link:configurator.asciidoc[configurator]).
+* *Maintainable* +
+For your project you should copy these link:settings.asciidoc[settings] to an own git repository that can be maintained and updated to manage the tool configurations during the project lifecycle. If you use GitHub or GitLab every developer can easily suggest changes and improvements to these link:settings.asciidoc[settings] via pull/merge requests, which is easier to manage with big teams.
+* *Customizable* +
+Do you need an additional tool you had never heard of before? Put it in the `software` folder of the link:structure.asciidoc[structure]. The `devon` link:cli.asciidoc[CLI] will then automatically add it to your `PATH` variable. +
+Further you can create your own link:cli.asciidoc#commandlets[commandlet] for your additional tool. For closed-source tools you can create your own archive and distribute it to your team members as long as you care about the terms and licenses of these tools.
+* *Multi-platform* +
+It works on all major platforms: Windows, Mac and Linux.
+* *Multi-tenancy* +
+You can have several instances of the `devonfw-ide` "link:setup.asciidoc[installed]" on your machine for different projects with different tools, tool versions and configurations. You won’t need to set up any physical installation nor changing your operating system. "Installations" of `devonfw-ide` do not interfere with each other nor with other installed software.
+* *Multiple Workspaces* +
+It supports working with different link:workspaces.asciidoc[workspaces] on different branches. You can create and update new workspaces with a few clicks. You can see the workspace name in the title-bar of your IDE so you do not get confused and work on the right branch.
+* *Free* +
+The `devonfw-ide` is free just like everything from https://devonfw.com[devonfw]. See link:license.asciidoc[LICENSE] for details.
+
+=== IDEs
+We support the following IDEs:
 
+* link:eclipse.asciidoc[Eclipse]
+* link:vscode.asciidoc[Visual Studio Code]
+* link:intellij.asciidoc[IntelliJ]
+
+=== Platforms
+We support the following platforms:
+
+* link:java.asciidoc[java] (see also https://github.com/devonfw/devon4j/wiki[devon4j])
+* link:cs.asciidoc[C#] (see https://devon4net.github.io/[devon4net])
+* link:node.asciidoc[node], link:ng.asciidoc[angular] and link:ionic.asciidoc[ionic] (see https://github.com/devonfw/devon4ng/wiki[devon4ng])
+
+=== Build-Systems
+We support the following build-systems:
```

Comment:
Also, this sounds like others are generally not supported, but these are the ones that are supported out of the box. Others are possible with manual effort.